res/config.json

```diff
@@ -3,18 +3,21 @@
     {
       "42": {
         "trunks": [
-          {
-            "42": {
-              "stations": [
-                {
-                  "endpoint": "SIP/100"
-                },
-                {
-                  "endpoint": "SIP/96@cwolfe"
-                }
-              ]
-            }
-          }
+          "42-A"
+        ],
+       "stations": [
+         "SIP/bobmarley",
+         "SIP/kingcrimson"
+        ]
+      }
+    },
+    {
+      "420": {
+        "trunks": [
+          "42-A"
+        ],
+       "stations": [
```

Comment:
Perhaps I'm using the wrong prettifier- http://jsonformatter.curiousconcept.com/ - but it looks the same as what I had originally.